doc/devices/remote.rst

```diff
@@ -4,17 +4,117 @@ The Remote device
 The ``'qiskit.remote'`` device is a generic adapter to use any Qiskit backend as interface
 for a PennyLane device.
 
-This device is useful when retrieving backends from providers with complex search options in
-their ``get_backend()`` method, or for setting options on a backend prior to wrapping it as
-PennyLane device.
+To access IBM backends, we recommend using `Qiskit Runtime <https://docs.quantum.ibm.com/api/migration-guides/qiskit-runtime-from-ibm-provider>`_.
 
 .. code-block:: python
 
-    import pennylane as qml
+    from qiskit_ibm_runtime import QiskitRuntimeService
 
-    def configured_backend():
-        backend = SomeProvider.get_backend(...)
-	backend.options.update_options(...)
-	return backend
+    QiskitRuntimeService.save_account(channel="ibm_quantum", token="<IQP_TOKEN>")
```

Comment:
Why do we call it `<IQP_>`? What is IQP an acronym for?

Reply:
IBM Quantum Platform (?) I think. It's copy and pasted from Qiskit's own documentation, but I could replace it with a more informative name like "your_ibm_q_token"

Reply:
It should be okay, I was just curious lol.